src/conic.jl

```diff
@@ -953,7 +953,7 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
                     inf_cut_generator = false
                     
                     # Update separator to primal solution
-                    separator = dcp_primal
+                    separator = copy(dcp_primal)
```

Comment:
Are all these copies necessary? There's some extra memory allocation that we could avoid.